app/services/streamlabels/index.ts

```diff
@@ -122,8 +124,6 @@ export class StreamlabelsService extends Service {
 
   init() {
     this.ensureDirectory();
-    this.fetchInitialData();
-    this.fetchSettings();
```

Comment:
also not sure why these are getting removed

Reply:
Both methods just exit if the user is not in logged-in  state. The user will never be in a logged-in state here. So these calls are useless now